README.md

```diff
@@ -50,11 +96,18 @@ The [docs](/docs) section contains details about each supported Terraform resour
 ### Release process
 
 1. Create a new branch from `master` as `release-vX.X.X`
+
    ```bash
    git branch release-vX.X.X master
    git push -u origin release-vX.X.X
    ```
-2. Update the `CHANGELOG.md` with your release version, date and change details and push the changes to the new branch. 
+
+2. Update the `CHANGELOG.md` with your release version, date and change details and push the changes to the new branch. New changes between previous release vY.Y.Y and current one vX.X.X can be found with the following command:
+
+   ```bash
+   git log --oneline vY.Y.Y..
```

Comment:
`git log --oneline vY.Y.Y..vX.X.X`

if vX.X.X is omitted, it will default to the current HEAD of the repo